package NYT.NChunkServer.NProto;

import "yt_proto/yt/core/misc/proto/guid.proto";

////////////////////////////////////////////////////////////////////////////////

message TReqLostVitalChunksSample
{
    required uint32 cell_tag = 1;
    repeated NYT.NProto.TGuid chunk_ids = 2;
}

////////////////////////////////////////////////////////////////////////////////
